5_People/career.md

```diff
@@ -22,41 +24,35 @@ This is the Salary Formula:
 
 > Note: all amounts are gross and per-month.
 
-### Base Salary
-
-Niteo Base Salary is calculated by first taking the [*Glassdoor salary for the role in San Francisco*](https://www.glassdoor.com/Salaries/san-francisco-software-engineer-salary-SRCH_IL.0,13_IM759_KO14,31.htm), converting it to EUR and finally discounting it by our *affordability ratio*. 
-
-#### Why San Francisco?
+#### Base Salary
 
-We take San Francisco because we assume it has the best data for the tech sector.
+Niteo Base Salary is calculated by first taking the [*Salary.com salary base for the role in United States*](https://www.salary.com/research), converting it to EUR and finally discounting it by our *affordability ratio*. 
```

Comment:
Not true, we no longer discount salaries by afforability ratio. The locationFactors are compressed towards affordability ratio. See https://github.com/niteoweb/salarycalc/pull/44/files#diff-0f1be8eec416e71c28b41efb0544df26R4